keps/sig-network/3178-iptables-cleanup/README.md

```diff
@@ -0,0 +1,764 @@
+  delayed if a commonly-used third-party component breaks badly when
+  the feature gate is enabled, but the SIG might choose not to delay
+  Beta for a bug involving a component which is not widely used, or
+  which can be worked around by upgrading to a newer version of that
+  component, or by changing its configuration.)
+
+#### GA
+
+- At least one release has passed since Beta
+
+- The SIG is not aware of any problems with third-party components
+  that would justify delaying GA. (For example, if the feature breaks
+  a third-party component which is no longer maintained and not likely
+  to ever be fixed, then there is no point in delaying GA because of
+  it.)
+
+### Upgrade / Downgrade Strategy
+
+Other than version skew (below), there are no upgrade / downgrade
+issues; kube-proxy recreates all of the rules it uses from scratch on
+startup, so for the purposes of this KEP there is no real difference
+between starting a fresh kube-proxy and upgrading an existing one.
+
+### Version Skew Strategy
+
+As long as we wait two releases between Alpha and Beta, then all
+allowed skewed versions of kubelet and kube-proxy will be compatible
+with each other.
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [X] Feature gate (also fill in valuesin `kep.yaml`)
+  - Feature gate name: IPTablesCleanup
+  - Components depending on the feature gate:
+    - kubelet
+
+###### Does enabling the feature change any default behavior?
+
+When the feature gate is enabled, kubelet will no longer create the
+IPTables chains/rules that it used to. This may cause problems with
+third-party components in Alpha but these problems are expected to be
+ironed out before moving to Beta.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+Nothing unexpected
+
+###### Are there any tests for feature enablement/disablement?
+
+No... there is no real difference between enabling the feature in an
+existing cluster vs creating a cluster where it was always enabled.
+
+### Rollout, Upgrade and Rollback Planning
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+The most likely cause of a rollout failure would be a third-party
+component that depended on one of the no-longer-existing IPTables
+chains. It is impossible to predict exactly how this third-party
+component would fail in this case, but it would likely impact already
+running workloads.
+
+###### What specific metrics should inform a rollback?
+
+Any failures would be the result of third-party components being
```

Comment:
I understand that we can't predict what some other tool will do. But perhaps we could give some hints as to where folks could look. For example, what components (CNI, netpol implementation, etc.) are likely to have potential errors or failures?